### DISCUSSION TITLE
have press on relay button open climate page

### DISCUSSION THREAD

**Discussion by iscraigh** [Blackymas/NSPanel_HA_Blueprint#1318] [Ideas]
Okay I cannot write an automation to save my life but how would I assign a relay button press to wake up the screen and  open  up the climate page, or is that even possible.  Could a press run an automation that does this?

Thanks
Craig

**Comment by iscraigh** [comment]
Would this work if I changed "home" to climate? I assume I add this to esphome.

```
binary_sensor:
  - id: !extend left_button
    on_click:
      - lambda: |-
          if (current_page->state == "screensaver") disp1->goto_page("home");
  - id: !extend right_button
    on_click:
      - lambda: |-
          if (current_page->state == "screensaver") disp1->goto_page("home");
```

Thanks

**Reply by edwardtfn** [reply]
It will probably work if you wanna see the main climate assigned to your panel. If you wanna show some other entity, then it should be a bit different.
